tests/common/dualtor/dual_tor_utils.py

```diff
@@ -11,6 +11,9 @@
 
 logger = logging.getLogger(__name__)
 
+UPPER_TOR = 'upper_tor'
+LOWER_TOR = 'lower_tor'
```

Comment:
Not blocking this PR.

can we define these at a shared place? (next file also have them defined).

Reply:
Sure. I'll address this in separate PR.